core/service/tests/backward_compatibility_kms.rs

```diff
@@ -152,9 +155,11 @@ fn test_kms_fhe_key_handles(
     let decompression_key: Option<DecompressionKey> =
         load_and_unversionize_auxiliary(dir, test, &test.decompression_key_filename, format)?;
 
+    let key_id = RequestId::zeros();
```

Comment:
Shouldn't this fail since the all zero requestId is invalid?

Reply:
I think we should decide what to do with the zero identifier. either remove the `zeros` function, or remove the check. I made an issue here: https://github.com/zama-ai/kms-internal/issues/2732